Pod/Classes/Heartbeat/SEGAdobeHeartbeat.m

```diff
@@ -5,4 +5,58 @@
 //  Created by ladan nasserian on 12/2/17.
 //
 
-#import <Foundation/Foundation.h>
+#import "SEGAdobeHeartbeat.h"
+#import <Analytics/SEGAnalytics.h>
+
+@interface VideoAnalyticsProvider : NSObject <ADBMediaHeartbeatDelegate>
```

Comment:
What is this? how is it being used?

Reply:
It will eventually have the QoS and getPlaybackPostion logic